indigo_api/migrations/0129_migrate_refs_uris.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-22 12:39
+from __future__ import unicode_literals
+
+from lxml import etree
+
+from django.db import migrations
+
+from cobalt import Act, FrbrUri, datestring
+from cobalt.uri import FRBR_URI_RE
+from indigo.analysis.refs.base import BaseRefsFinder
+
+
+def new_frbr_uri(uri, forward):
+    """ Generates a cobalt FrbrUri object using all the existing URI's attributes.
+        If `forward` is True, the optional /akn prefix is included; otherwise it isn't.
+    """
+    if not isinstance(uri, FrbrUri):
+        uri = FrbrUri.parse(uri)
+    prefix = None
+    if forward:
+        prefix = 'akn'
+
+    return FrbrUri(
```

Comment:
You can also use `frbr_uri = frbr_uri.clone(); frbr_uri.prefix = 'xxx'`

Reply:
I don't even think you need to generate a new object, you can simply set the prefix on the existing object.